install.sh

```diff
@@ -712,16 +712,16 @@ additional_shellenv_commands=()
 if [[ "${HOMEBREW_BREW_DEFAULT_GIT_REMOTE}" != "${HOMEBREW_BREW_GIT_REMOTE}" ]]
 then
   ohai "HOMEBREW_BREW_GIT_REMOTE is set to a non-default URL:"
-  echo "${tty_underline}${HOMEBREW_BREW_GIT_REMOTE}${tty_reset} will be used for Homebrew/brew Git remote."
+  echo "${tty_underline}${HOMEBREW_BREW_GIT_REMOTE}${tty_reset} will be used as the Homebrew/brew Git remote."
   non_default_repos="Homebrew/brew"
   additional_shellenv_commands+=("export HOMEBREW_BREW_GIT_REMOTE=\"${HOMEBREW_BREW_GIT_REMOTE}\"")
 fi
 
 if [[ "${HOMEBREW_CORE_DEFAULT_GIT_REMOTE}" != "${HOMEBREW_CORE_GIT_REMOTE}" ]]
 then
   ohai "HOMEBREW_CORE_GIT_REMOTE is set to a non-default URL:"
-  echo "${tty_underline}${HOMEBREW_CORE_GIT_REMOTE}${tty_reset} will be used for Homebrew/core Git remote."
-  non_default_repos="${non_default_repos:-}${non_default_repos:+ and }Homebrew/core"
```

Comment:
Similiar for Homebrew/core:

https://github.com/Homebrew/brew/blob/aaea979a428a004a9a681e92deecc122320c5a48/Library/Homebrew/cmd/update.sh#L514-L516

https://github.com/Homebrew/brew/blob/5c44aca2e197fcaf0d1377257686e3eea7710405/Library/Homebrew/tap.rb#L788-L790

Reply:
These too.